### PR TITLE
[Merged by Bors] - chore: Some constants in `CanonicalEmbedding` should be `NNReal` and not `ENNReal`

### PR DESCRIPTION
**Pull request by xroblot**
---

- [x] depends on: #9837 

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

**Review by jcommelin — APPROVED** [review]
Thanks :tada:

bors merge

**Comment by Vierkantor** [comment]
I think this PR is causing the build failures. Let me try merging master on my machine to see if it breaks...

**Comment by Vierkantor** [comment]
Indeed this seems to be out of date with the master branch.

bors r-

bors d+

File: Mathlib/NumberTheory/NumberField/Discriminant.lean [hunk 1]
```diff
@@ -110,10 +110,8 @@ theorem exists_ne_zero_mem_ideal_of_norm_le_mul_sqrt_discr (I : (FractionalIdeal
     refine le_of_eq ?_
     rw [convexBodySum_volume, ← ENNReal.ofReal_pow (by positivity), ← Real.rpow_nat_cast,
       ← Real.rpow_mul toReal_nonneg, div_mul_cancel, Real.rpow_one, ofReal_toReal, mul_comm,
-      mul_assoc, ENNReal.inv_mul_cancel (convexBodySumFactor_ne_zero K)
-      (convexBodySumFactor_ne_top K), mul_one]
-    · exact mul_ne_top (ne_of_lt (minkowskiBound_lt_top K I))
-        (ENNReal.inv_ne_top.mpr (convexBodySumFactor_ne_zero K))
+      mul_assoc, ← coe_mul, inv_mul_cancel (convexBodySumFactor_ne_zero K), coe_one, mul_one]
```
Comment (Vierkantor) [inline Mathlib/NumberTheory/NumberField/Discriminant.lean:113]:
```suggestion
      mul_assoc, ← coe_mul, inv_mul_cancel (convexBodySumFactor_ne_zero K), ENNReal.coe_one,
      mul_one]
```

**Pull request closed without merging**